### ISSUE TITLE
Loading metainfo doesn't work

### ISSUE THREAD

**Issue by ERamseth** [libasys/pinit#3]
entering a URL and trying to load meta info doesn't work. It just keeps "loading" forever.

Using OC 8.1.0 and pinit 1.0.1

(this was also reported by someone else on apps.owncloud.org)
##

**Comment by libasys** [comment]
Can you give me the url where have you tried? Thanks!

**Comment by ERamseth** [comment]
I tried http://ddg.gg and http://player.siriusxm.com and http://core4x4.org

**Comment by ERamseth** [comment]
I've found that various youtube videos actually do work. Maybe the sites I pointed out in the above don't pass the proper info when you query them? In that case you should do something other than perpetually wait for it to load.

**Comment by libasys** [comment]
Yeah showing metainfo works, but i only showing a result if a image was found! But I have changed this behaviour with version 1.0.2! Now you can grab metainfo without image found!
